packages/scanner/test/cli/commands.spec.ts

```diff
@@ -49,6 +52,8 @@ describe('commands', () => {
           let watcher: watchScan.Watcher | undefined;
           jest.spyOn(watchScan, 'default').mockImplementation((opts) => {
             watcher = new watchScan.Watcher(opts);
+            clearTimeout(watcher.interval);
+            watcher.interval.unref(); // else jest doesn't exit
```

Comment:
With `unref` in the `Watcher` constructor we can omit these in each test.

Reply:
Removed these `.unref` calls from each test.